client/src/utils/utils.ts

```diff
@@ -119,90 +120,26 @@ export function textify(lst) {
  * @param{Function} error   - Callback on error
  * @param{Boolean}  cache   - Use cached data if available
  */
-export function get(options) {
-    top.__utils__get__ = top.__utils__get__ || {};
-    var cache_key = JSON.stringify(options);
-    if (options.cache && top.__utils__get__[cache_key]) {
-        if (options.success) {
-            options.success(top.__utils__get__[cache_key]);
-        }
-        window.console.debug(`utils.js::get() - Fetching from cache [${options.url}].`);
-    } else {
-        request({
-            url: options.url,
-            data: options.data,
-            success: function (response) {
-                top.__utils__get__[cache_key] = response;
-                if (options.success) {
-                    options.success(response);
-                }
-            },
-            error: function (response, status) {
-                if (options.error) {
-                    options.error(response, status);
-                }
-            },
-        });
-    }
+interface getOptions {
+    url: string;
+    data?: any;
+    success?: (data: AxiosResponse) => void;
+    error?: (error: AxiosError) => void;
```

Comment:
Interface conflicts with jsdoc comment

Reply:
Thanks for catching this; I updated the docstring.  Cache is no longer supported (and wasn't used anywhere), and data was previously undocumented.